.github/workflows/dart_ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Dart CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [stable, dev]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dart-lang/setup-dart@v0.1
+        with:
+          channel: ${{ matrix.sdk }}
+
+      - name: Print Dart SDK version
+        run: dart --version
+
+      - id: install
+        name: Install dependencies
+        run: dart pub get
+
+      - name: Validate dependencies
+        run: dart pub run dependency_validator -i build_runner,build_test,build_web_compilers
+        if: always() && steps.install.outcome == 'success'
+
+      # TODO: Uncomment this, and remove the Dart 2.7.2 format step in the Workiva Build Dockerfile
+      # once we are able to bump the lower bound of SDK.
```

Comment:
We don't need to keep formatting in Dart 2.7.2; before my Dockerfile PR, we had been formatting on stable (it doesn't matter to me tohugh)

Reply:
I think most contributors are running 2.7.x - so we should keep formatting that way IMO.